templates/resources/provisioner.md.tmpl

```diff
@@ -0,0 +1,28 @@
+---
+Title: "Provisioner resources in a management cluster"
+Description: |-
+  Fetching provisioner resources in a management cluster.
```

Comment:
It's not just fetching right? I would be provisioner lifecycle!!?

Reply:
Yes, updated